lib/animina_web/live/profile_photo_live.ex

```diff
@@ -115,7 +115,7 @@ defmodule AniminaWeb.ProfilePhotoLive do
         params = Map.merge(params, file)
         form = Form.validate(socket.assigns.form, params, errors: true)
 
-        Photo.destroy(socket.assigns.current_user.profile_photo)
+        Photo.delete_photo_and_optimized_photos(socket.assigns.current_user.profile_photo)
```

Comment:
If you were new to this project and wanted to delete a `Photo`, would you use `destroy` or `delete_photo_and_optimized_photos`? Rewrite the `destroy` action so that we can use it here as it feels natural. We `destroy` a `Photo` and the resource takes care of all the details.